spring-cloud-gcp-docs/src/main/asciidoc/sql.adoc

```diff
@@ -47,7 +49,9 @@ public List<Map<String, Object>> listUsers() {
 }
 ----
 
-`JdbcTemplate` is configured based on the following properties:
+This starter is configured through
+https://github.com/spring-projects/spring-boot/blob/v1.5.8.RELEASE/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jdbc/DataSourceProperties.java[`DataSourceProperties`].
```

Comment:
I would make it through the JavaDocs link: https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/jdbc/DataSourceProperties.html.

At least that way we don't have any version mentioning to worry about in the future